.github/workflows/validate-json.yaml

```diff
@@ -0,0 +1,29 @@
+# Validate JSON schema files
+# Source: https://github.com/nhalstead/validate-json-action
+# Forked from: https://github.com/marketplace/actions/validate-json
+name: Validate JSON schemas for assets and materials
+
+on: 
+  push:
+    branches: [ main ]
+
+jobs:
+  json-schema-validation:
+    name: JSON schema validation
+    runs-on: ubuntu-latest
+    
+    steps:
+      - uses: actions/checkout@v1
+      
+      - name: Validate "asset" JSON schemas
+        uses: nhalstead/validate-json-action@0.1.3
+        with:
+          schema: ./schemas/asset_schema.json
+          jsons: ./examples/*_asset.xoma
```

Comment:
Remove "_asset" from the filter, since "example_asset" is just the file name. Filtering for the extension should be enough.